extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -114,6 +114,16 @@ function process_anchor_params() {
 		}
 	}
 
+	// Display an outbound link after publishing a post (only to English-speaking users since Anchor
+	// is English only).
+	if (
```

Comment:
Understanding it's a beta block at the moment, I'm still wondering if we should add a filter in here as our feature gate (that way we can turn it on on WP.com based on the dev blog sticker – one less thing to troubleshoot).

`apply_filters( 'jetpack_anchor_fm_dev', false )`

Reply:
Since the block is in Beta, I think you should be able to use the existing Beta blog sticker (`jetpack-beta-blocks`) if you want to test this when non-proxied on specific blocks.

Reply:
Yup, exactly. Beta blocks are enabled by default for Automatticians who are proxied and the existing `jetpack-beta-blocks` blog sticker should cover the rest while the block is still in beta.

Reply:
Ah great! 👍